src/notifications/generate.ts

```diff
@@ -65,7 +65,7 @@ export const notificationTitleMap: Record<
   squad_member_joined: (
     ctx: NotificationSourceContext & NotificationDoneByContext,
   ) =>
-    `Your squad is growing! <b>${ctx.doneBy.name}</b> has <span class="text-theme-color-cabbage">joined</span> <b>${ctx.source.name}</b>.`,
+    `Your squad <b>${ctx.source.name}</b> is <span class="text-theme-color-cabbage">growing</span>! Welcome <b>${ctx.doneBy.name}</b> to the squad with an upvote or comment.`,
```

Comment:
Are you sure the copy is `with an upvote or comment`? I can't welcome with an upvote. Worth checking with product

Reply:
Copy mentions: `Your squad {{{squad_name}}} is growing.  Welcome {{{new username}}} to your squad with a comment.`

Reply:
Oh, I referred to the Sketch file. Will verify which is the latest.

![Screenshot 2023-05-12 at 12 33 55 PM](https://github.com/dailydotdev/daily-api/assets/13744167/38b930ec-3ffa-4db8-a2a9-c1881267c97e)